src/core/App/Store.js

```diff
@@ -0,0 +1,16 @@
+import popularMoviesReducer from "../../features/MovieList/popularMoviesSlice";
+import rootSaga from "../App/rootSaga";
+import createSagaMiddleware from "@redux-saga/core";
+import { configureStore } from "@reduxjs/toolkit";
+
+const sagaMiddleware = createSagaMiddleware();
+const store = configureStore({
+  reducer: {},
```

Comment:
I don't think we need this empty reducer. Every new reducers we will create we can put together.